### PR TITLE
Fix TLS communication issue

### PR DESCRIPTION
**Pull request by suyogpatil**
Solves https://github.com/stackrox/admission-controller-webhook-demo/issues/8

### CODE REVIEW

**Comment by keymone** [comment]
@misberner ping, merge please, otherwise this repo just doesn't work

**Comment by shravanshetty1** [comment]
LGTM

**Review by misberner — APPROVED** [review]

File: deployment/deployment.yaml.template [hunk 1]
```diff
@@ -15,9 +15,6 @@ spec:
       labels:
         app: webhook-server
     spec:
-      securityContext:
```
Comment (misberner) [inline deployment/deployment.yaml.template:18]: @suyogpatil why did you remove this? Typically, we do not want a webhook server to run as root

**Comment by misberner** [comment]
Sorry for the long delay. I'll merge as soon as the qustion wrt the security context is resolved

File: deployment/deployment.yaml.template (hunk 1 above)
Comment (suyogpatil) [reply deployment/deployment.yaml.template:18]: I remember that after adding this securityContext webhook or some related container was not able to create..its been long time so dont exactly remember but it caused some probelm afaik

Comment (misberner) [reply deployment/deployment.yaml.template:18]: Hmm.. I just tested with this, and it works. Do you mind removing this from the PR for now? If it causes issues, we can remove it again, with a proper bug description attached.

Comment (suyogpatil) [reply deployment/deployment.yaml.template]: I removed it. Thanks

**Review by misberner — APPROVED** [review]
Thanks, @suyogpatil !
